test/e2e/v4/plugin_cluster_test.go

```diff
@@ -412,8 +412,23 @@ func Run(kbc *utils.TestContext, hasWebhook, isToUseInstaller, isToUseHelmChart,
 		_, err = kbc.Kubectl.Apply(true, "-f", conversionCRPath)
 		Expect(err).NotTo(HaveOccurred(), "failed to apply modified ConversionTest CR")
 
-		// TODO: Add validation to check the conversion
+		// Apply validation to check the conversion
 		// the v2 should have spec.replicas == 3
+		By("validating that v2 spec.replicas is set to 3 via conversion")
+		Eventually(func(g Gomega) {
+			output, err := kbc.Kubectl.Get(
+				true,
+				"-n", "test-ns",
```

Comment:
I need to re-check the code above to confirm think the ns is `true` only right?

Reply:
Yes. And I have added validation check for v1 -> v2 conversion. Thanks

Reply:
I think we should not `"-n", "test-ns",`
The true will get the CRs from the namespace defined in the TestContext
Give a look on its implementation

Reply:
Yes, I found that.

Do I need establish a CRD for that? sth like
```
crdName := "conversiontests." + kbc.Group
Eventually(func(g Gomega) {
    _, err := kbc.Kubectl.Get(false, "crd", crdName)
    g.Expect(err).NotTo(HaveOccurred(), "CRD %s not found", crdName)
}
```

Reply:
@p-shah256 currently I am working here, plz free to review it. thanks